ci/test_suites/authenticators_k8s/test_gke_entrypoint.sh

```diff
@@ -97,6 +104,29 @@ function finish {
   delete_image "$NGINX_TAG"
   set -e
 }
+
+function get_pod_names(){
+    kubectl get pods -o json \
+    |jq -r '.items[].metadata.name'
+}
+
+function get_pod_container_names(){
+  pod="${1}"
+  kubectl  get "pod/${pod}" -o json \
+    |jq -r '.spec.containers[].name'
+}
+
+function dump_pod_logs(){
```

Comment:
It looks like we're already dumping the Conjur containers logs, starting on line 38 of this file. Could we consolidate these approaches so Conjur's logs are only dumped once?

Reply:
Otherwise LGTM

Reply:
Currently the Conjur logs are always printed, I'd rather not change that format. Would it be ok to just print all the other logs on an error, omitting Conjur as it is already printed?